### DISCUSSION TITLE
Stake FET tokens in the native FET chain

### DISCUSSION THREAD

**Discussion by Wdmoore01** [ping-pub/explorer#562] [Q&A]
What wallet can we use to stake FET tokens in the native FET chain? (Not the Eth chain)  I have swapped FET tokens in Kraken form SNX to Fetch.ai in the Fet Native chain. So I am looking for a web wallet that I can send the tokens to so I can Stake them, using the native FET chain. I have spent a lot of time checking google searches and have not found a current one to allow me to stake my tokens. I was told Keplr, and Metamask, and Trust wallets. None of which support htis token presently. And I am not using a Ledger Cold wallet.

**Comment by liangping** [comment]
Sorry. I don't have FET.  but I think Keplr should work
